frontend/heir/backends/openfhe/backend.py

```diff
@@ -0,0 +1,227 @@
+            options=(heir_opt_options),
+        )
+        if(debug):
+            # Print output after heir_opt:
+            mlirpath = Path(workspace_dir) / f"{func_name}.backend.mlir"
+            graphpath = Path(workspace_dir) / f"{func_name}.backend.dot"
+            print(f"HEIRpy Debug (OpenFHE Backend): Writing backend MLIR to {mlirpath}")
+            with open(mlirpath, "w") as f:
+                f.write(heir_opt_output)
+            print(f"HEIRpy Debug (OpenFHE Backend): Writing backend graph to {graphpath}")
+            with open(graphpath, "w") as f:
+                f.write(graph)
+
+
+        # Translate to *.cpp and Pybind
+        module_name = f"_heir_{func_name}"
+        cpp_filepath = Path(workspace_dir) / f"{func_name}.cpp"
+        h_filepath = Path(workspace_dir) / f"{func_name}.h"
+        pybind_filepath = Path(workspace_dir) / f"{func_name}_bindings.cpp"
+        include_type_flag = "--openfhe-include-type=" + self.openfhe_config.include_type
+        header_options = [
+                "--emit-openfhe-pke-header",
+                include_type_flag,
+                "-o",
+                h_filepath,
+            ]
+        cpp_options = ["--emit-openfhe-pke", include_type_flag, "-o", cpp_filepath]
+        pybind_options =  [
+                "--emit-openfhe-pke-pybind",
+                f"--pybind-header-include={h_filepath.name}",
+                f"--pybind-module-name={module_name}",
+                "-o",
+                pybind_filepath,
+            ]
+        if(debug):
+            print("HEIRpy Debug (OpenFHE Backend): " + Style.BRIGHT + f"Running heir-translate {' '.join(str(o) for o in header_options)}")
+        heir_translate.run_binary(
+            input=heir_opt_output,
+            options=header_options,
+        )
+        if(debug):
+            print("HEIRpy Debug (OpenFHE Backend): " + Style.BRIGHT + f"Running heir-translate {' '.join(str(o) for o in cpp_options)}")
+        heir_translate.run_binary(
+            input=heir_opt_output,
+            options=cpp_options,
+        )
+        if(debug):
+            print("HEIRpy Debug (OpenFHE Backend): " + Style.BRIGHT + f"Running heir-translate {' '.join(str(o) for o in pybind_options)}")
+        heir_translate.run_binary(
+            input=heir_opt_output,
+            options=pybind_options,
+        )
+
+        clang_backend = clang.ClangBackend()
+        so_filepath = Path(workspace_dir) / f"{func_name}.so"
+        linker_search_paths = [self.openfhe_config.lib_dir]
+        if(debug):
+            args = clang_backend.clang_arg_helper(cpp_source_filepath=cpp_filepath,
+            shared_object_output_filepath=so_filepath,
+            include_paths=self.openfhe_config.include_dirs,
+            linker_search_paths=linker_search_paths,
+            link_libs=self.openfhe_config.link_libs)
+            print(
+            f"HEIRpy Debug (OpenFHE Backend):\033[1m Running clang {' '.join(str(arg) for arg in args)}\033[0m"
```

Comment:
```suggestion
            f"HEIRpy Debug (OpenFHE Backend):" + Style.BRIGHT + f" Running clang {' '.join(str(arg) for arg in args)}"
```

Guessing this is what you had before using colorama?

Reply:
Ditto below on line 186